paper/basic_training.tex

```diff
@@ -1030,15 +1031,15 @@ \subsection{Long range electrostatics}
 \subsubsection{Motivation}
 
 The calculation of non-bonded interactions is generally the most time-consuming step of classical energy calculation.
-While the number of type of bonded interactions remain unchanged during an MD simulation, the strength and importance of non-bonded interactions varies substantially as a simulation proceeds. 
+While the number of type of bonded interactions remain unchanged during an MD simulation, the strength and importance of non-bonded interactions varies substantially as a simulation proceeds.
 
 Additionally, Coulombic interactions fall off only very slowly with distance, as $r^{-1} $, further complicating handling of non-bonded interactions in two different ways.
-First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order of operations~\cite{LeachBook}, meaning we must exercise extreme care.
+First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order in which it is evaluated~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.
```

Comment:
```suggestion
First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order \emph{in which it is evaluated}~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.
```

Seems like a pretty important point to make. Might be worthwhile to italicize it.

Reply:
OK, I'll fix in my other PR (Heather merged this one).